### DISCUSSION TITLE
Unity에서 Play On Zepeto 오류

### DISCUSSION THREAD

**Discussion by njngwn** [naverz/zepeto-studio-kor#855] [5. Q&A]
안녕하세요. 유니티에서 월드를 제작하던 중, 중간테 스트를 진행하고자 play on zepeto로 QR 코드를 생성해 스마트폰을 통해 제페토 앱을 실행하였으나,

계속해서 보라색 화면만 보이더니, 
'제페토 월드 접속에 실패했어요. 다시 시도해 주세요.'라는 메세지가 뜨네요.

url로도 시도해봤는데, 마찬가지입니다. 계속해서 시도해봤는데 안되네요.
처음 빌드할때 큐브 돌아가는 건 잘 되었던 걸로 기억하는데 혹시 제가 뭘 놓쳤는지 알고 계신 분 있으실까요?

감사합니다.

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 테스트하시는 PC와 모바일 기기가 동일한 네트워크에 연결되었는지 확인 부탁드립니다.

**Reply by njngwn** [reply]
답변 감사드립니다. 현재 동일한 네트워크 사용 중입니다. 여전히 안되네요...
